recipe/meta.yaml

```diff
@@ -1,62 +1,71 @@
-{% set version = "2.3.8" %}
+{% set version = "5.6.0" %}
 
 package:
   name: pymc
   version: {{ version }}
 
 source:
   url: https://github.com/pymc-devs/pymc/archive/v{{version}}.tar.gz
-  sha256: 200a28469d574a370fa9b55cd92f112f8ad09f71674bdae514437039e953689b
+  sha256: a09b11a0ae9a56a6f57b9b89f161d06b1d49b7d80a84311eba4376f2a32b5d0f
 
 build:
   number: 0
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
   script:
     - export LDFLAGS="-undefined dynamic_lookup -bundle"  # [osx]
     - export LDFLAGS="$LDFLAGS -shared"  # [linux]
-    - "python setup.py install"
+    - python -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('fortran') }}
   host:
     - python
-    - mkl-devel    {{ mkl }}
-    - numpy-devel  {{ numpy }}  # [not (osx and arm64) and py<=39]
-    - numpy {{ numpy }}  # [(osx and arm64) or py>39]
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - scipy
-    # - matplotlib
-    - numpy-devel  {{ numpy }}  # [not (osx and arm64) and py<=39]
-    - numpy {{ numpy }}         # [(osx and arm64) or py>39]
+    - numpy >=1.15.0
+    - arviz >=0.13.0
+    - cachetools >=4.2.1
+    - cloudpickle
+    - fastprogress >=0.2.0
+    - pandas >=0.24.0
+    - pytensor >=2.12.0,<2.13
+    - scipy >=1.4.1
+    - typing-extensions >=3.7.4
 
 test:
   imports:
     - pymc
-    - pymc.database
-    - pymc.examples
+    - pymc.backends
+    - pymc.distributions
     - pymc.gp
-    - pymc.gp.cov_funs
-    - pymc.Container_values
-    - pymc.LazyFunction
-    - pymc.flib
-    - pymc.gp.cov_funs.distances
-    - pymc.gp.cov_funs.isotropic_cov_funs
-    - pymc.gp.incomplete_chol
-    - pymc.gp.linalg_utils
+    - pymc.step_methods
+    - pymc.tuning
+    - pymc.variational
+    - pymc.ode
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-  home: http://github.com/pymc-devs/pymc
+  home: https://www.pymc.io/welcome.html
   license: AFL-3.0
   license_family: OTHER
```

Comment:
Should not it be "APACHE" here? I think AFL refers to Apache?

Reply:
right, thanks for catching this. I fixed the license as it is Apache-2.0.